docs/misc/math-expr.md

```diff
@@ -115,53 +115,53 @@ See javadoc of java.lang.Math for detailed explanation for each function.
-|bitwiseComplement|bitwiseComplement(x) would return the result of ~x. Double values will be implicitly cast to longs, use `bitwiseConvertDoubleToLongBits` to perform bitwise operations directly with doubles|
-|bitwiseConvertDoubleToLongBits|bitwiseConvertDoubleToLongBits(x) would convert the IEEE 754 floating-point "double" bits stored in a long into a double value if the input is a long, or implicitly cast the value to a long if the input is a double|
-|bitwiseConvertLongBitsToDouble|bitwiseConvertLongBitsToDouble(x) would convert a long to the IEEE 754 floating-point "double" specified by the bits stored in the long. A double input will be implicitly cast to a long|
-|bitwiseOr|bitwiseOr(x,y) would return the result of x [PIPE] y. Double values will be implicitly cast to longs, use `bitwiseConvertDoubleToLongBits` to perform bitwise operations directly with doubles|
-|bitwiseShiftLeft|bitwiseShiftLeft(x,y) would return the result of x << y. Double values will be implicitly cast to longs, use `bitwiseConvertDoubleToLongBits` to perform bitwise operations directly with doubles|
-|bitwiseShiftRight|bitwiseShiftRight(x,y) would return the result of x >> y. Double values will be implicitly cast to longs, use `bitwiseConvertDoubleToLongBits` to perform bitwise operations directly with doubles|
-|bitwiseXor|bitwiseXor(x,y) would return the result of x ^ y. Double values will be implicitly cast to longs, use `bitwiseConvertDoubleToLongBits` to perform bitwise operations directly with doubles|
-|atan2|atan2(y, x) would return the angle theta from the conversion of rectangular coordinates (x, y) to polar * coordinates (r, theta)|
-|cbrt|cbrt(x) would return the cube root of x|
-|ceil|ceil(x) would return the smallest (closest to negative infinity) double value that is greater than or equal to x and is equal to a mathematical integer|
-|copysign|copysign(x) would return the first floating-point argument with the sign of the second floating-point argument|
-|cos|cos(x) would return the trigonometric cosine of x|
-|cosh|cosh(x) would return the hyperbolic cosine of x|
-|cot|cot(x) would return the trigonometric cotangent of an angle x|
+|abs|abs(x) returns the absolute value of x|
+|acos|acos(x) returns the arc cosine of x|
+|asin|asin(x) returns the arc sine of x|
+|atan|atan(x) returns the arc tangent of x|
+|bitwiseAnd|bitwiseAnd(x,y) returns the result of x & y. Double values will be implicitly cast to longs, use `bitwiseConvertDoubleToLongBits` to perform bitwise operations directly with doubles|
+|bitwiseComplement|bitwiseComplement(x) returns the result of ~x. Double values will be implicitly cast to longs, use `bitwiseConvertDoubleToLongBits` to perform bitwise operations directly with doubles|
+|bitwiseConvertDoubleToLongBits|bitwiseConvertDoubleToLongBits(x) will convert the IEEE 754 floating-point "double" bits of a double value into a long, or implicitly cast the value to a double if the input is a long.|
```

Comment:
This still doesn't seem right; surely, if the input is a long, the function will cast the value to a double and _then_ convert those double bits back to a long. The description doesn't make it sound like that's what happens.